### PR TITLE
Add Deprecation notice to all functions that have been moved to the Glaze CLI

### PR DESCRIPTION
**Pull request by Sterahi**
## Deprecate all functions that have been transitioned over to the Glaze CLI.

We are now providing an alert after most functions that looks like the following: 
 
Screen Shot 2022-01-12 at 11 24 49 AM

The updated help looks like the following: 
Screen Shot 2022-01-12 at 11 24 05 AM

## How Has This Been Tested?

The deprecation notice has been run multiple times to ensure that it behaves properly & ANSI escapes were used to ensure no issues with colors/different Operating Systems.

## PR checklist

Before submitting this PR, please make sure:

- [x] I have tagged the relevant reviewers and interested parties
- [x] I have updated the READMEs of affected packages
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by PaulLeCam — COMMENTED** [review]
LGTM but I'm not familiar with how well formatting encoding is supported by different OSes and shells, aren't there differences to take into accounts or at least support checks?

**Comment by ukstv** [comment]
Also, raw colour-chars output could potentially break on a terminal, that does not support colors. So, it maybe is a good idea to use `picocolors` or `kleur` for colors: they both have a decent function to determine if a terminal supports colors.

**Comment by Sterahi** [comment]
@ukstv Implemented picocolors. thanks for pointing it out! :)

File: packages/cli/package.json [hunk 1]
```diff
@@ -59,7 +59,8 @@
     "levelup": "^4.4.0",
     "morgan": "^1.10.0",
     "nft-did-resolver": "^1.0.0",
-    "pkh-did-resolver": "^0.3.8-rc.0",
+    "picocolors": "^1.0.0",
+    "pkh-did-resolver": "^0.3.3",
```
Comment (ukstv) [inline packages/cli/package.json:63]: @Sterahi mind I ask you, please, why pkh-did-resolver version has changed?

Comment (Sterahi) [reply packages/cli/package.json]: I have no idea, I've reset it to it's previous iteration though!

**Review by ukstv — APPROVED** [review]
🏇

**Review by stbrody — APPROVED** [review]
lgtm, thanks!

**Pull request merged**